attachment_uploader.rb

```diff
@@ -0,0 +1,50 @@
+class AttachmentUploader < CarrierWave::Uploader::Base
```

Comment:
It is told [here](https://github.com/carrierwaveuploader/carrierwave) that yes, we need it.